uvloop/loop.pyx

```diff
@@ -89,6 +89,22 @@ cdef inline socket_dec_io_ref(sock):
         sock._decref_socketios()
 
 
+cdef inline run_in_context(context, method):
+    # This method is internally used to workaround a reference issue that in
```

Comment:
My understanding here is that `method` is going to be increfed by cython even if `run_in_context` is inlined? Should we add explicit INCREF/DECREF here for clarity?

Reply:
> `method` is going to be increfed by cython even if `run_in_context` is inlined?

Yes, Cython is always creating a new reference and `GOTREF` before calling the inlined function, and `DECREF` afterwards, unless the `method` is already a caller's local reference.

> Should we add explicit INCREF/DECREF here for clarity?

I'm okay either way - as this is likely a temporary workaround depending on if we should fix Cython or CPython, upon which I'd appreciate some light shed.

Reply:
Let's add explicit incref/decref, the code will be way more clear that way. The performance penalty is going to be negligible. While it's temporary, we don't know how temporary it will be. If we want to fix Cython it can be an issue for a few months (although this doesn't look like a Cython issue, after all, right?). And if we want to fix CPython, it will take years before we stop supporting 3.9.1.